src/targets/services/categorize.js

```diff
@@ -66,11 +66,19 @@ export const categorize = async () => {
   )
 
   // Categorize each doc and update it
+  const categoriesBefore = {}
+  const categoriesAfter = {}
   const categorized = operations.map(operation => {
     const prediction = model.predict(operation.label)
+
+    categoriesBefore[operation.cozyCategoryId || '0'] =
+      (categoriesBefore[operation.cozyCategoryId || '0'] || 0) + 1 // Default to uncategorized
+    categoriesAfter[prediction || '0'] =
+      (categoriesAfter[prediction || '0'] || 0) + 1 // Default to uncategorized
+
     return {
       ...operation,
-      automaticCategoryId: prediction
+      cozyCategoryId: prediction
```

Comment:
We should change to cozyCategoryId everywhere, by doing a quick search, automaticCategoryId is used elsewhere